.github/workflows/test-pipeline.yml

```
@@ -379,176 +64,17 @@
+        run: npm audit --audit-level moderate || true
       
       - name: CodeQL Analysis
         uses: github/codeql-action/init@v3
         with:
           languages: typescript
           queries: security-and-quality
       
-      - name: Build for CodeQL
+      - name: Build for analysis
         run: npm run build:fast
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)